dashboard/src/index.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.remove-animate-pulse.animate-pulse {
+  animation: none !important;
+}
```

Comment:
why are you adding this?

Reply:
Because `animate-pulse` is in the original component, so I created this class that cancels out the previous one.